.gitignore

```diff
@@ -1,12 +1,13 @@
 CFHT/
 DECam/
-input/
-output/
+CFHT
+DECam
 test.log
-bin/validateCfht.py
-bin/validateDecam.py
+*.json
```

Comment:
Ah, this was a bit forward-looking to DM-5121, which will create JSON output files.